src/main/java/fi/tkgwf/ruuvi/config/Config.java

```diff
@@ -146,15 +143,34 @@ private static void readConfig() {
         }
     }
 
-    private static void readTagNames() {
-        try {
-            File jarLocation = new File(Config.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath()).getParentFile();
-            File[] configFiles = jarLocation.listFiles(f -> f.isFile() && f.getName().equals("ruuvi-names.properties"));
+    private static File[] findConfigFiles(final String propertiesFileName) throws URISyntaxException {
```

Comment:
Since there will always be 0 or 1 matching config files, returning an array is a bit redundant considering all use-cases just pick the first entry, so the return could be changed to a plain File rather than File[].